main_routers/workshop_router.py

```
@@ -0,0 +1,1161 @@
+            "error": "Steamworks未初始化",
+            "message": "请确保Steam客户端已运行且已登录"
+        }, status_code=503)
+    
+    query_handle = None
+    try:
+        item_id_int = int(item_id)
+        item_state = steamworks.Workshop.GetItemState(item_id_int)
+        query_handle = steamworks.Workshop.CreateQueryUGCDetailsRequest([item_id_int])
+        
+        # Define a callback for UGC query completion
+        def ugc_query_callback(result):
```

Ruff (0.14.8)

Unused function argument: `result`

(ARG001)